addon/templates/components/nypr-accounts/membership-card/index.hbs

```diff
@@ -0,0 +1,156 @@
+                  <div class="pledge-order-cc-type">{{pledge.creditCardType}} {{pledge.creditCardLast4Digits}}</div>
+                </div>
+              </div>
+              <div class="pledge-update-link">
+                <a href="https://pledge3.wnyc.org/donate/membership-sustainer/sustainer/sustainer?order_id={{pledge.orderKey}}">Update</a>
+              </div>
+            </div>
+          {{/each}}
+
+        {{else if activeOneTimePledges}}
+          <div class="pledge-active-onetime-member">
+            <p class="last-pledge-date">You last donated on <span class="pledge-date">{{moment-format mostRecentPledge.orderDate 'MMMM D, YYYY'}}</span>.</p>
+            <a class="pledge-donate-button-link" href="https://pledge3.wnyc.org/donate/membership-sustainer/sustainer/?order_id={{mostRecentPledge.orderKey}}" target="_blank">
+              <p class="pledge-donate-button">Make another donation</p>
+            </a>
+          </div>
+
+        {{else if isLapsedOneTimeMember}}
+          <div class="pledge-active-onetime-member">
+            <p class="last-pledge-date">It looks like you haven’t given since
+              {{moment-format mostRecentPledge.orderDate 'MMM. D, YYYY'}}.  Take
+              a moment to renew your membership right now!</p>
+            <a class="pledge-donate-button-link"
+                href="https://pledge3.wnyc.org/donate/membership-sustainer/sustainer/?order_id={{mostRecentPledge.orderKey}}"
+                target="_blank">
+              <p class="pledge-donate-button">Donate to renew</p>
+            </a>
+          </div>
+
+        {{else if isLapsedSustainer}}
+          <div class="pledge-lapsed-sustainer">
+            <p class="last-pledge-date">Your sustaining donations have expired.</p>
+            <a class="pledge-donate-button-link" href="https://pledge3.wnyc.org/donate/membership-sustainer/sustainer/" target="_blank">
+              <p class="pledge-donate-button">Become a member</p>
+            </a>
+          </div>
+
+        {{else}} {{! i.e. isNonMember }}
+          <div class="pledge-non-member">
+            <p class="last-pledge-date">Our records show you're not a member yet. How about changing that right now, and help pay for all of the great programming you hear on WNYC.</p>
+            <a class="pledge-donate-button-link" href="https://pledge3.wnyc.org/donate/membership-sustainer/sustainer/" target="_blank">
+              <p class="pledge-donate-button">Become a member</p>
+            </a>
+          </div>
+        {{/if}}
+      </div>
+    {{/if}}
+
+    <div class="pledge-tools">
+      {{#if pledges}}
+      <div class="pledge-payment-history"><span {{action (action (mut showPaymentHistory) (not showPaymentHistory))}}>Giving History</span></div>
+      {{/if}}
+      <div class="pledge-help-link"><span {{action (action (mut disableModal) true)}}>Help</span></div>
+
+    </div>
+  {{/nypr-card}}
+
+  {{#if disableModal}}
```

Comment:
FWIW I'm matching the behavior of the "Disable my account" modal from web client https://github.com/nypublicradio/wnyc-web-client/blob/b8ffc2e33149633f2ecfc2b8633058a623f7d38c/app/templates/profile.hbs#L30